charts/quickwit/values.yaml

```diff
@@ -178,24 +196,30 @@ metastore:
 
   # Extra env for metastore
   extraEnv: {}
-    # KEY: VALUE
+  # KEY: VALUE
   # This is the recommended way to inject `QW_METASTORE_URI` when using the postgres metastore (see https://quickwit.io/docs/configuration/metastore-config)
   extraEnvFrom: []
     # - secretRef:
     #     name: quickwit-metastore
     # - configMapRef:
-    #     name: quickwit-metastore
+  #     name: quickwit-metastore
+
+  # extraVolumes -- Additional Volumes to use with Vector Pods.
+  extraVolumes: []
+
+  # extraVolumeMounts -- Additional Volume to mount into Vector Containers.
+  extraVolumeMounts: []
 
   resources: {}
     # limits:
     #   cpu: 100m
     #   memory: 128Mi
     # requests:
     #   cpu: 100m
-    #   memory: 128Mi
+  #   memory: 128Mi
 
   updateStrategy: {}
-    # type: RollingUpdate
+  # type: RollingUpdate
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed